package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.html",
   "type": "module",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "format": "npx prettier --write .",
+    "lint": "npx eslint --fix ."
```

Comment:
Since `eslint` was already in the repository, I added a command to make use of it and added `prettier`, with its config, so that all files were formatted in a standard way.